src/Card.js

```diff
@@ -0,0 +1,313 @@
+import Phaser from 'phaser'
+import { boardDrag, cardDimensions, hoverButtonRadius, cardBackFrame, cardDepth, activeDepth, hoverOffset, canvasHeight, inHandAdjustment, canvasWidth, inHandRange } from './Constants'
+export default class Card extends Phaser.GameObjects.Container {
+  constructor(scene, x, y, physicsGroup, cardNumber, orientation = Math.PI/2) {
+    super(scene, x, y)
+    //add images to container
+    this.shadow = scene.add.image(0,0,'shadow')
+    this.card = scene.add.sprite(0,0,'cardSprite')
+    this.card.setFrame(cardBackFrame)
+    this.flipButton = scene.add.image(cardDimensions.width/2 - hoverButtonRadius + hoverOffset, hoverButtonRadius - cardDimensions.height/2 - hoverOffset,'flip').setVisible(false)
```

Comment:
This line is super long, it might be worth breaking up the variables within it to make it more readable (Same for the lines below it)